### ISSUE TITLE
invalid element state: Element is not currently interactable and may not be manipulated

### ISSUE THREAD

**Issue by tegger2000** [Second-Hand-Friends/kleinanzeigen-bot#75]
Hi, I get the following error almost every time I try
```
[ERROR] Traceback (most recent call last):
  File "/usr/local/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/tegger/Downloads/kleinanzeigen-bot/kleinanzeigen_bot/__main__.py", line 8, in <module>
    kleinanzeigen_bot.main(sys.argv)
  File "/home/tegger/Downloads/kleinanzeigen-bot/kleinanzeigen_bot/__init__.py", line 605, in main
    KleinanzeigenBot().run(args)
  File "/home/tegger/Downloads/kleinanzeigen-bot/kleinanzeigen_bot/__init__.py", line 77, in run
    self.publish_ads(ads)
  File "/home/tegger/Downloads/kleinanzeigen-bot/kleinanzeigen_bot/__init__.py", line 381, in publish_ads
    self.publish_ad(ad_file, ad_cfg, ad_cfg_orig)
  File "/home/tegger/Downloads/kleinanzeigen-bot/kleinanzeigen_bot/__init__.py", line 453, in publish_ad
    self.web_input(By.ID, "pstad-street", ad_cfg["contact"]["street"])
  File "/home/tegger/Downloads/kleinanzeigen-bot/kleinanzeigen_bot/selenium_mixin.py", line 305, in web_input
    input_field.clear()
  File "/home/tegger/Downloads/kleinanzeigen-bot/__pypackages__/3.10/lib/selenium/webdriver/remote/webelement.py", line 93, in clear
    self._execute(Command.CLEAR_ELEMENT)
  File "/home/tegger/Downloads/kleinanzeigen-bot/__pypackages__/3.10/lib/selenium/webdriver/remote/webelement.py", line 710, in _execute
    return self._parent.execute(command, params)
  File "/home/tegger/Downloads/kleinanzeigen-bot/__pypackages__/3.10/lib/selenium/webdriver/remote/webdriver.py", line 425, in execute
    self.error_handler.check_response(response)
  File "/home/tegger/Downloads/kleinanzeigen-bot/__pypackages__/3.10/lib/selenium/webdriver/remote/errorhandler.py", line 247, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidElementStateException: Message: invalid element state: Element is not currently interactable and may not be manipulated
  (Session info: chrome=90.0.4430.212)
Stacktrace:
#0 0x55591421a7f9 <unknown>
#1 0x5559141ba3b3 <unknown>
#2 0x555913f02016 <unknown>
#3 0x555913f05659 <unknown>
#4 0x555913f0548e <unknown>
#5 0x555913f05714 <unknown>
#6 0x555913f2fa63 <unknown>
#7 0x555913f59dc2 <unknown>
#8 0x555913f2dd7b <unknown>
#9 0x555913f59fce <unknown>
#10 0x555913f6a9e1 <unknown>
#11 0x555913f59c93 <unknown>
#12 0x555913f2bce4 <unknown>
#13 0x555913f2d4d2 <unknown>
#14 0x5559141e6542 <unknown>
#15 0x5559141f5ce7 <unknown>
#16 0x5559141f59e4 <unknown>
#17 0x5559141fa13a <unknown>
#18 0x5559141f65b9 <unknown>
#19 0x5559141dbe00 <unknown>
#20 0x55591420d5d2 <unknown>
#21 0x55591420d778 <unknown>
#22 0x555914225a1f <unknown>
#23 0x7f05a7e8ffa3 start_thread
#24 0x7f05a70e5eff clone
```

```

**Comment by sebthom** [comment]
Seems to be a  duplicate of #74

**Comment by sebthom** [comment]
Fixed by #79. Thanks to @DenisLanz
